### PR TITLE
Fixes #437: Added Swagger UI

### LINKED ISSUE #437: Add Swagger UI
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**
Swagger UI is not present

**Expected behavior:**
As this PR: https://github.com/fossasia/badgeyay/commit/82d6fcd79b087ba96d6cf4c20c1d075ed50a9c97 merged  
We need to add Swagger UI. Swagger does a simple but very powerful thing: with a bit of additional annotations it generates the REST API descriptions (HTTP methods, path / query / form parameters, responses, HTTP error codes, ...) and even provides a simple web UI to play with REST calls to your APIs (not to mention that all this metadata is available over REST as well).

**Yes, I want to work on the issue**

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
Fixes #437 

#### Checklist
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:
- Automaticaly add documentation based on the schema declared in the function
- Documentation serves at: http://127.0.0.1:5000/apidocs/

Reference Screenshot:
image

image

image

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
@djmgit @gabru-md @ParthS007 @yashLadha Please review

**Comment by harshit98** [comment]
@vaibhavsingh97 Kindly resolve conflicts so that I can merge this PR. Cheers!

**Comment by djmgit** [comment]
@vaibhavsingh97 please update this PR.

**Comment by vaibhavsingh97** [comment]
@harshit98 @djmgit PLease review and merge. Fixed the conflicts

Sorry for delay as internet was not working due to maintainance

**Comment by vaibhavsingh97** [comment]
@djmgit please merge

**Comment by mariobehling** [comment]
@vaibhavsingh97 Usually, we request two reviews of co-developers before it can get merged. Please work with others to resolve blockers asap. Conflicts are there again. Please resolve. Thank you.

File: backend/migrations/script.py.mako [hunk 1]
```diff
@@ -0,0 +1,24 @@
+"""${message}
```
Comment (yashLadha) [inline backend/migrations/script.py.mako]: These files are already present in the api folder. __Duplicate__ it is. @vaibhavsingh97

File: backend/migrations/alembic.ini [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# A generic, single database configuration.
```
Comment (yashLadha) [inline backend/migrations/alembic.ini]: Same here

File: backend/migrations/env.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
+from __future__ import with_statement
```
Comment (yashLadha) [inline backend/migrations/env.py]: Same as above

File: backend/migrations/versions/e4a611919da8_.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
+"""empty message
```
Comment (yashLadha) [inline backend/migrations/versions/e4a611919da8_.py]: Same as above

**Comment by vaibhavsingh97** [comment]
@gabru-md @ParthS007 @yashLadha @djmgit Please review it again

**Comment by djmgit** [comment]
@ParthS007 @yashLadha @gabru-md please review again.

**Comment by djmgit** [comment]
ping @ParthS007 @gabru-md @yashLadha

**Comment by djmgit** [comment]
@ParthS007 @yashLadha @gabru-md please have look at this PR as @vaibhavsingh97 has updated it. Waiting for your reviews.

**Comment by djmgit** [comment]
@vaibhavsingh97 tarvis is failing.

**Comment by vaibhavsingh97** [comment]
@djmgit Travis never failed 👍 , Actually someone triggered the Travis and canceled so it shows like that 😄

**Pull request merged**